app/src/main/res/values/strings.xml

```diff
@@ -123,4 +123,8 @@
     <string name="zero">0</string>
     <string name="name_is_empty">Name is Empty</string>
     <string name="description_is_empty">Description is Empty</string>
+    <string name="create_shop_successful">Create shop successful!</string>
+    <string name="please_wait">Please Wait..</string>
+    <string name="loading">Loading…</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.loading` appears to be unused
 Unused resources make applications larger and slow down builds.